core/src/main/scala/org/apache/spark/sql/delta/storage/ClosableIterator.scala

```diff
@@ -25,19 +25,19 @@ object ClosableIterator {
    * An implicit class for applying a function to a [[ClosableIterator]] and returning the
    * resulting iterator as a [[ClosableIterator]] with the original `close()` method.
    */
-  implicit class IteratorCloseOps[A](val iterator: ClosableIterator[A]) extends AnyVal {
+  implicit class IteratorCloseOps[A](val closableIter: ClosableIterator[A]) extends AnyVal {
     def withClose[B](f: Iterator[A] => Iterator[B]): ClosableIterator[B] = new ClosableIterator[B] {
       private val iter =
         try {
-          f(iterator)
```

Comment:
The change is because https://github.com/scala/scala-collection-compat/pull/7 changed `iterator()` to `iterator` and it conflicts with the `IteratorCloseOps.iterator` field. Renaming it to fix it for 2.13.